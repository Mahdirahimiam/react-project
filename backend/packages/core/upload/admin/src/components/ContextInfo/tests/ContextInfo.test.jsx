import React from 'react';

import { lightTheme, ThemeProvider } from '@strapi/design-system';
import { render } from '@testing-library/react';

import { ContextInfo } from '../index';

describe('ContextInfo', () => {
  it('renders', () => {
    const { container } = render(
      <ThemeProvider theme={lightTheme}>
        <ContextInfo
          blocks={[
            {
              label: 'Label 1',
              value: 'value1',
            },

            {
              label: 'Label 2',
              value: 'value2',
            },

            {
              label: 'Label 3',
              value: 'value3',
            },
          ]}
        />
      </ThemeProvider>
    );

    expect(container).toMatchSnapshot();
  });
});
