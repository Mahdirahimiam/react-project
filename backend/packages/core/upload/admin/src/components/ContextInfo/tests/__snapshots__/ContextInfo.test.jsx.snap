// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContextInfo renders 1`] = `
.c6 {
  border: 0;
  -webkit-clip: rect(0 0 0 0);
  clip: rect(0 0 0 0);
  height: 1px;
  margin: -1px;
  overflow: hidden;
  padding: 0;
  position: absolute;
  width: 1px;
}

.c4 {
  font-weight: 600;
  font-size: 0.6875rem;
  line-height: 1.45;
  text-transform: uppercase;
  color: #666687;
}

.c5 {
  font-size: 0.75rem;
  line-height: 1.33;
  color: #4a4a6a;
}

.c0 {
  background: #f6f6f9;
  padding-top: 16px;
  padding-right: 24px;
  padding-bottom: 16px;
  padding-left: 24px;
  border-radius: 4px;
}

.c3 {
  -webkit-align-items: stretch;
  -webkit-box-align: stretch;
  -ms-flex-align: stretch;
  align-items: stretch;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  gap: 4px;
}

.c1 {
  display: grid;
  grid-template-columns: repeat(12,1fr);
  gap: 16px;
}

.c2 {
  grid-column: span 6;
  max-width: 100%;
}

@media (max-width:68.75rem) {
  .c2 {
    grid-column: span;
  }
}

@media (max-width:34.375rem) {
  .c2 {
    grid-column: span 12;
  }
}

<div>
  <div
    class="c0"
  >
    <div
      class="c1"
    >
      <div
        class="c2"
      >
        <div
          class="c3"
        >
          <span
            class="c4"
          >
            Label 1
          </span>
          <span
            class="c5"
          >
            value1
          </span>
        </div>
      </div>
      <div
        class="c2"
      >
        <div
          class="c3"
        >
          <span
            class="c4"
          >
            Label 2
          </span>
          <span
            class="c5"
          >
            value2
          </span>
        </div>
      </div>
      <div
        class="c2"
      >
        <div
          class="c3"
        >
          <span
            class="c4"
          >
            Label 3
          </span>
          <span
            class="c5"
          >
            value3
          </span>
        </div>
      </div>
    </div>
  </div>
  <div
    class="c6"
  >
    <p
      aria-live="polite"
      aria-relevant="all"
      id="live-region-log"
      role="log"
    />
    <p
      aria-live="polite"
      aria-relevant="all"
      id="live-region-status"
      role="status"
    />
    <p
      aria-live="assertive"
      aria-relevant="all"
      id="live-region-alert"
      role="alert"
    />
  </div>
</div>
`;
